examples/Gallery/src/AddAutoScroll/Gallery.tsx

```diff
@@ -10,15 +10,17 @@ export const Gallery = (props: { pictureData: Picture[] }) => {
   const galleryRef = useRef<NodesRef>(null);
 
   useEffect(() => {
-    galleryRef.current
-      ?.invoke({
-        method: "autoScroll",
-        params: {
-          rate: "60",
-          start: true,
-        },
-      })
-      .exec();
+    setTimeout(() => {
```

Comment:
why do you need this `setTimeout`?

Reply:
It shouldn't be; it's been fixed in @lynx-js/web-elements@0.8.10